meta-dts-distro/recipes-dts/dts/dts/dts

```diff
@@ -153,15 +169,17 @@ while : ; do
       fi
       ;;
     2)
-      if check_network_connection; then
-        echo "Preparing ..."
-        if [ -z "${logs_sent}" ]; then
-          export SEND_LOGS="true"
-          if ! ${CMD_DASHARO_HCL_REPORT} > /dev/null 2>&1; then
-            echo -e "Cannot reach 3mdeb cloud to leave HCL report. Please
-                      \rrecheck your internet connection."
-          else
-            logs_sent="1"
+      if ! look_for_dasharo; then
+        if check_network_connection; then
+          echo "Preparing ..."
+          if [ -z "${logs_sent}" ]; then
+            export SEND_LOGS="true"
```

Comment:
@esesyn01 this would need to be changed after we change the function name

Reply:
@TomaszAIR Done - https://github.com/Dasharo/meta-dts/pull/25/commits/22b1c2c1ec48f722fcfd8901d10439bc1b9ecf4b#diff-d2bd602c4c837aa03dd0dd7eb140ddf24397c7c33b1961b4b83d04f014010d64R157